app/javascript/src/Common/application.ts

```diff
@@ -0,0 +1,63 @@
+import type { JQueryXHR } from 'Types/jquery/v1'
+
+const jQuery1 = window.$
+
+export default function (): void {
+  // disable links with 'data-disabled' attribute and display alert instead
+  // delegation on body fires before rails.js. FIXME: this is not a valid guard. If "data-disabled"
+  // removed manually from the DOM, the request will still go through. The link should be disabled
+  // (not clickable) and the server should return a flash error.
+  $('body').on('click', 'a[data-disabled]', ({ currentTarget, stopImmediatePropagation }) => {
+    stopImmediatePropagation()
```

Comment:
The original code also had `preventDefault`. So, it's not needed?

Reply:
It's already covered by `return false`. HOWEVER, after further investigation it turns out returning a value was deprecated and is recommended not to use it (even though most browsers supports it). So I will revert it.